src/map/MapSidebar.js

```diff
@@ -97,6 +126,138 @@ function MapSidebarMain (props) {
+  // gets the date of the next Meetup
+  useEffect(() => {
+    async function getNextMeetupDate(){
+      const meetupLocations = await api.rpc.encointer.getLocations(cid);
+      const tempLocation = locationFromJson(api, meetupLocations[0]);
+      const temporaryTime = await getNextMeetupTime(api, tempLocation);
+      const tempdate = new Date(temporaryTime.toNumber());
+      const temparray = (tempdate.toString()).split("G");
+      const tempresultarray = temparray[0].split(" ");
+      setNextMeetupTime (" "+tempresultarray[0]+" "+tempresultarray[2]+" "+tempresultarray[1]+ " "+tempresultarray[3]);
+    }
+    getNextMeetupDate();
+  }, [api,nextMeetupTime, setNextMeetupTime, cid]);
+
+  // gets the nominal Income of a Community
+  const [nominalIncome, setNominalIncome]= useState([]);
+  useEffect(() => {
+    async function getNominalIncome(){
+      const tempNominalIncome = await getCeremonyIncome(api, cid);
+      return parseI64F64(tempNominalIncome);
+    }
+    let isMountedNominal = true 
+    getNominalIncome().then((data) => {
+      if (isMountedNominal) setNominalIncome(" "+data);
+    });
+    return () => {isMountedNominal = false; };
+
+  },[nominalIncome, setNominalIncome,getCeremonyIncome, api, cid])
+
+  const [registeredBootstrappers, setregisteredBootstrappers]= useState([]);
+  const [registeredReputables, setregisteredReputables]= useState([]);
+  const [registeredEndorseees, setregisteredEndorsees]= useState([]);
+  const [registeredNewbies, setregisteredNewbies]= useState([]);
+  const [unassignedNewbies, setunassignedNewbies]= useState([]);
+  
+// gets the data of all people that registered for a Ceremony
+  function showPhasepeopleData(){
+    useEffect(() => {
+      async function getPhasepeopleData(){
+        const currentCeremonyIndex = await api.query.encointerScheduler.currentCeremonyIndex();
+        const currentCommunityCeremony = new CommunityCeremony(api.registry, [cid, currentCeremonyIndex])
+        const [tempregisteredBootstrappers, tempregisteredReputables, tempregisteredEndorseees,tempregisteredNewbies, assignmentCounts] = await Promise.all([
+          api.query.encointerCeremonies.bootstrapperCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.reputableCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.endorseeCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.newbieCount(currentCommunityCeremony),
+          api.query.encointerCeremonies.assignmentCounts(currentCommunityCeremony)
+        ]);
+        return [tempregisteredBootstrappers, tempregisteredReputables, tempregisteredEndorseees, tempregisteredNewbies, assignmentCounts]
+      }
+
+      let isMounted = true;
+      getPhasepeopleData().then((data) => {
+        if(isMounted){
+        setregisteredBootstrappers(data[0]);
+        setregisteredReputables(data[1]);
+        setregisteredEndorsees(data[2]);
+        setregisteredNewbies(data[3]);
```

Comment:
Minor detail here. Here we need to distinguis between two cases:

* Registering phase: use `data[i]` for the fields.
* Assignment or Attestation phase use `asssignmentCount` fields for the displayed data. Then we should also change the text to: `Assigned Participants` instead of `Registered Participants`.

Unassigned newbies does only make sen

Reply:
I can put everything in an array which saves some room, but i cannot use data[i] for that I would need to use a useState const variable.
And I don't really get what you mean with using assignmentCount fields for the displayed data.

Reply:
I think it might be best, when we look at this some time together